### DISCUSSION TITLE
Uvicorn.run() not working in uvicorn v0.18

### DISCUSSION THREAD

**Discussion by Shugaba-Wuta** [Kludex/uvicorn#1540] [Potential Issue] [answered]
Hey, 
I was working with *FastAPI* and  *Uvicorn* , then I upgraded uvicorn to v0.18.1. After upgrading uvicorn, I discovered that I could not start uvicorn using the  *uvicorn.run()*  method which I works well with version *<=0.17.6*.  I should mention that the uvicorn works well with the CLI. 
Python version: 3.10.5; 
Windows 10; 
Virtualenv 20.14.1;
Fastapi 0.78.0; 

Here is a simple code snippet for investigations: 
```
import uvicorn
from fastapi import FastAPI

app = FastAPI() 
@app.get("/")
def index(): 
    return {"home": "index"}

if __name__==  "__main__": 
    uvicorn.run("main:app", reload=True)
```

**Comment by Kludex** [comment, marked as answer]
It's working. The logs are not being sent. Thanks for reporting it.

Fix is on the way: https://github.com/encode/uvicorn/pull/1541
